CRITICAL - 2024-05-01 00:27:53 --> You have not selected a database type to connect to.
in SYSTEMPATH\Database\Database.php on line 53.
 1 SYSTEMPATH\Database\Config.php(82): CodeIgniter\Database\Database->load([...], 'default')
 2 SYSTEMPATH\Common.php(362): CodeIgniter\Database\Config::connect('default', true)
 3 APPPATH\Controllers\Home.php(56): db_connect()
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->terminos()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-05-01 00:30:58 --> You have not selected a database type to connect to.
in SYSTEMPATH\Database\Database.php on line 53.
 1 SYSTEMPATH\Database\Config.php(82): CodeIgniter\Database\Database->load([...], 'default')
 2 SYSTEMPATH\Common.php(362): CodeIgniter\Database\Config::connect('default', true)
 3 APPPATH\Controllers\Home.php(56): db_connect()
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->terminos()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-05-01 00:31:13 --> You have not selected a database type to connect to.
in SYSTEMPATH\Database\Database.php on line 53.
 1 SYSTEMPATH\Database\Config.php(82): CodeIgniter\Database\Database->load([...], 'default')
 2 SYSTEMPATH\Common.php(362): CodeIgniter\Database\Config::connect('default', true)
 3 APPPATH\Controllers\Home.php(56): db_connect()
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->terminos()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-05-01 00:32:22 --> You have not selected a database type to connect to.
in SYSTEMPATH\Database\Database.php on line 53.
 1 SYSTEMPATH\Database\Config.php(82): CodeIgniter\Database\Database->load([...], 'default')
 2 SYSTEMPATH\Common.php(362): CodeIgniter\Database\Config::connect('default', true)
 3 APPPATH\Controllers\Home.php(56): db_connect()
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->terminos()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-05-01 00:32:23 --> You have not selected a database type to connect to.
in SYSTEMPATH\Database\Database.php on line 53.
 1 SYSTEMPATH\Database\Config.php(82): CodeIgniter\Database\Database->load([...], 'default')
 2 SYSTEMPATH\Common.php(362): CodeIgniter\Database\Config::connect('default', true)
 3 APPPATH\Controllers\Home.php(56): db_connect()
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->terminos()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-05-01 00:32:47 --> You have not selected a database type to connect to.
in SYSTEMPATH\Database\Database.php on line 53.
 1 SYSTEMPATH\Database\Config.php(82): CodeIgniter\Database\Database->load([...], 'default')
 2 SYSTEMPATH\Common.php(362): CodeIgniter\Database\Config::connect('default', true)
 3 APPPATH\Controllers\Home.php(56): db_connect()
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->terminos()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-05-01 00:32:48 --> You have not selected a database type to connect to.
in SYSTEMPATH\Database\Database.php on line 53.
 1 SYSTEMPATH\Database\Config.php(82): CodeIgniter\Database\Database->load([...], 'default')
 2 SYSTEMPATH\Common.php(362): CodeIgniter\Database\Config::connect('default', true)
 3 APPPATH\Controllers\Home.php(56): db_connect()
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->terminos()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-05-01 00:32:48 --> You have not selected a database type to connect to.
in SYSTEMPATH\Database\Database.php on line 53.
 1 SYSTEMPATH\Database\Config.php(82): CodeIgniter\Database\Database->load([...], 'default')
 2 SYSTEMPATH\Common.php(362): CodeIgniter\Database\Config::connect('default', true)
 3 APPPATH\Controllers\Home.php(56): db_connect()
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->terminos()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-05-01 00:32:49 --> You have not selected a database type to connect to.
in SYSTEMPATH\Database\Database.php on line 53.
 1 SYSTEMPATH\Database\Config.php(82): CodeIgniter\Database\Database->load([...], 'default')
 2 SYSTEMPATH\Common.php(362): CodeIgniter\Database\Config::connect('default', true)
 3 APPPATH\Controllers\Home.php(56): db_connect()
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->terminos()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-05-01 00:33:15 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\fastcars\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\fastcars\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\xampp\htdocs\fastcars\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\fastcars\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\fastcars\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\fastcars\app\Controllers\Home.php(65): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(934): App\Controllers\Home->terminos()
#6 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\fastcars\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\fastcars\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\fastcars\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\fastcars\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\fastcars\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\xampp\htdocs\fastcars\app\Controllers\Home.php(65): CodeIgniter\Database\BaseBuilder->insert(Array)
#4 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(934): App\Controllers\Home->terminos()
#5 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\fastcars\index.php(67): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2024-05-01 00:33:15 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombe`, `apellido`, `email`, `usuario`, `pass`) VALUES (:nombe:, :apellido:, :email:, :usuario:, :pass:)', [...], false)
 3 APPPATH\Controllers\Home.php(65): CodeIgniter\Database\BaseBuilder->insert([...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->terminos()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-05-01 00:34:55 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\fastcars\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\fastcars\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\xampp\htdocs\fastcars\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\fastcars\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\fastcars\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\fastcars\app\Controllers\Home.php(65): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(934): App\Controllers\Home->terminos()
#6 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\fastcars\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\fastcars\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\fastcars\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\fastcars\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\fastcars\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\xampp\htdocs\fastcars\app\Controllers\Home.php(65): CodeIgniter\Database\BaseBuilder->insert(Array)
#4 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(934): App\Controllers\Home->terminos()
#5 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\fastcars\index.php(67): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2024-05-01 00:34:55 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombe`, `apellido`, `email`, `usuario`, `pass`) VALUES (:nombe:, :apellido:, :email:, :usuario:, :pass:)', [...], false)
 3 APPPATH\Controllers\Home.php(65): CodeIgniter\Database\BaseBuilder->insert([...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->terminos()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-05-01 00:36:23 --> mysqli_sql_exception: Unknown column 'nombe' in 'field list' in C:\xampp\htdocs\fastcars\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\fastcars\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\fastcars\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\fastcars\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\fastcars\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\fastcars\app\Controllers\Home.php(65): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(934): App\Controllers\Home->terminos()
#6 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\fastcars\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2024-05-01 00:36:23 --> Unknown column 'nombe' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombe`, `apellido`, `email`, `usuario`, `pass`) VALUES (:nombe:, :apellido:, :email:, :usuario:, :pass:)', [...], false)
 2 APPPATH\Controllers\Home.php(65): CodeIgniter\Database\BaseBuilder->insert([...])
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->terminos()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-05-01 00:53:40 --> mysqli_sql_exception: Duplicate entry '0' for key 'PRIMARY' in C:\xampp\htdocs\fastcars\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\fastcars\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\fastcars\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\fastcars\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\fastcars\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\fastcars\app\Controllers\Home.php(65): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(934): App\Controllers\Home->terminos()
#6 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\fastcars\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2024-05-01 00:53:40 --> Duplicate entry '0' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `apellido`, `email`, `usuario`, `pass`) VALUES (:nombre:, :apellido:, :email:, :usuario:, :pass:)', [...], false)
 2 APPPATH\Controllers\Home.php(65): CodeIgniter\Database\BaseBuilder->insert([...])
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->terminos()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-05-01 01:03:11 --> mysqli_sql_exception: Duplicate entry 'francobarrientos56@gmail.com' for key 'email' in C:\xampp\htdocs\fastcars\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\fastcars\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\fastcars\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\fastcars\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\fastcars\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\fastcars\app\Controllers\Home.php(65): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(934): App\Controllers\Home->terminos()
#6 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\fastcars\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2024-05-01 01:03:11 --> Duplicate entry 'francobarrientos56@gmail.com' for key 'email'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `apellido`, `email`, `usuario`, `pass`) VALUES (:nombre:, :apellido:, :email:, :usuario:, :pass:)', [...], false)
 2 APPPATH\Controllers\Home.php(65): CodeIgniter\Database\BaseBuilder->insert([...])
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->terminos()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-05-01 01:03:12 --> mysqli_sql_exception: Duplicate entry 'francobarrientos56@gmail.com' for key 'email' in C:\xampp\htdocs\fastcars\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\fastcars\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\fastcars\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\fastcars\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\fastcars\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\fastcars\app\Controllers\Home.php(65): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(934): App\Controllers\Home->terminos()
#6 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\fastcars\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2024-05-01 01:03:12 --> Duplicate entry 'francobarrientos56@gmail.com' for key 'email'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `apellido`, `email`, `usuario`, `pass`) VALUES (:nombre:, :apellido:, :email:, :usuario:, :pass:)', [...], false)
 2 APPPATH\Controllers\Home.php(65): CodeIgniter\Database\BaseBuilder->insert([...])
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->terminos()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-05-01 01:03:13 --> mysqli_sql_exception: Duplicate entry 'francobarrientos56@gmail.com' for key 'email' in C:\xampp\htdocs\fastcars\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\fastcars\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\fastcars\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\fastcars\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\fastcars\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\fastcars\app\Controllers\Home.php(65): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(934): App\Controllers\Home->terminos()
#6 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\fastcars\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2024-05-01 01:03:13 --> Duplicate entry 'francobarrientos56@gmail.com' for key 'email'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `apellido`, `email`, `usuario`, `pass`) VALUES (:nombre:, :apellido:, :email:, :usuario:, :pass:)', [...], false)
 2 APPPATH\Controllers\Home.php(65): CodeIgniter\Database\BaseBuilder->insert([...])
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->terminos()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-05-01 01:03:50 --> mysqli_sql_exception: Duplicate entry 'fran123' for key 'usuario' in C:\xampp\htdocs\fastcars\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\fastcars\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\fastcars\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\fastcars\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\fastcars\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\fastcars\app\Controllers\Home.php(65): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(934): App\Controllers\Home->terminos()
#6 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\fastcars\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2024-05-01 01:03:50 --> Duplicate entry 'fran123' for key 'usuario'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `apellido`, `email`, `usuario`, `pass`) VALUES (:nombre:, :apellido:, :email:, :usuario:, :pass:)', [...], false)
 2 APPPATH\Controllers\Home.php(65): CodeIgniter\Database\BaseBuilder->insert([...])
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->terminos()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-05-01 01:03:52 --> mysqli_sql_exception: Duplicate entry 'fran123' for key 'usuario' in C:\xampp\htdocs\fastcars\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\fastcars\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\fastcars\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\fastcars\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\fastcars\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\fastcars\app\Controllers\Home.php(65): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(934): App\Controllers\Home->terminos()
#6 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\fastcars\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2024-05-01 01:03:52 --> Duplicate entry 'fran123' for key 'usuario'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `apellido`, `email`, `usuario`, `pass`) VALUES (:nombre:, :apellido:, :email:, :usuario:, :pass:)', [...], false)
 2 APPPATH\Controllers\Home.php(65): CodeIgniter\Database\BaseBuilder->insert([...])
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->terminos()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-05-01 01:04:49 --> mysqli_sql_exception: Duplicate entry 'fran123' for key 'usuario' in C:\xampp\htdocs\fastcars\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\fastcars\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\fastcars\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\fastcars\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\fastcars\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\fastcars\app\Controllers\Home.php(65): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(934): App\Controllers\Home->terminos()
#6 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\fastcars\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2024-05-01 01:04:49 --> Duplicate entry 'fran123' for key 'usuario'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `apellido`, `email`, `usuario`, `pass`) VALUES (:nombre:, :apellido:, :email:, :usuario:, :pass:)', [...], false)
 2 APPPATH\Controllers\Home.php(65): CodeIgniter\Database\BaseBuilder->insert([...])
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->terminos()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-05-01 01:04:51 --> mysqli_sql_exception: Duplicate entry 'fran123' for key 'usuario' in C:\xampp\htdocs\fastcars\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\fastcars\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\fastcars\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\fastcars\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\fastcars\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\fastcars\app\Controllers\Home.php(65): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(934): App\Controllers\Home->terminos()
#6 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\fastcars\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2024-05-01 01:04:51 --> Duplicate entry 'fran123' for key 'usuario'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `apellido`, `email`, `usuario`, `pass`) VALUES (:nombre:, :apellido:, :email:, :usuario:, :pass:)', [...], false)
 2 APPPATH\Controllers\Home.php(65): CodeIgniter\Database\BaseBuilder->insert([...])
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->terminos()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-05-01 01:16:42 --> mysqli_sql_exception: Table 'bd_barrientos_franco.usuario' doesn't exist in C:\xampp\htdocs\fastcars\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\fastcars\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\fastcars\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\fastcars\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\fastcars\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\fastcars\app\Models\Usuario.php(28): CodeIgniter\Database\BaseBuilder->insert(Object(App\Models\Usuario))
#5 C:\xampp\htdocs\fastcars\app\Controllers\Home.php(67): App\Models\Usuario->createUsuario()
#6 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(934): App\Controllers\Home->terminos()
#7 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\fastcars\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-05-01 01:16:42 --> Table 'bd_barrientos_franco.usuario' doesn't exist
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuario` (`id`, `nombre`, `apellido`, `usuario`, `email`, `pass`, `perfil_id`, `baja`, `pager`) VALUES (:id:, :nombre:, :apellido:, :usuario:, :email:, :pass:, :perfil_id:, :baja:, :pager:)', [...], false)
 2 APPPATH\Models\Usuario.php(28): CodeIgniter\Database\BaseBuilder->insert(Object(App\Models\Usuario))
 3 APPPATH\Controllers\Home.php(67): App\Models\Usuario->createUsuario()
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->terminos()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-05-01 01:17:24 --> mysqli_sql_exception: Unknown column 'pager' in 'field list' in C:\xampp\htdocs\fastcars\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\fastcars\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\fastcars\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\fastcars\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\fastcars\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\fastcars\app\Models\Usuario.php(28): CodeIgniter\Database\BaseBuilder->insert(Object(App\Models\Usuario))
#5 C:\xampp\htdocs\fastcars\app\Controllers\Home.php(67): App\Models\Usuario->createUsuario()
#6 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(934): App\Controllers\Home->terminos()
#7 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp\htdocs\fastcars\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\fastcars\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-05-01 01:17:24 --> Unknown column 'pager' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`id`, `nombre`, `apellido`, `usuario`, `email`, `pass`, `perfil_id`, `baja`, `pager`) VALUES (:id:, :nombre:, :apellido:, :usuario:, :email:, :pass:, :perfil_id:, :baja:, :pager:)', [...], false)
 2 APPPATH\Models\Usuario.php(28): CodeIgniter\Database\BaseBuilder->insert(Object(App\Models\Usuario))
 3 APPPATH\Controllers\Home.php(67): App\Models\Usuario->createUsuario()
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->terminos()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
